### ISSUE TITLE
Crash error

### ISSUE THREAD

**Issue by youngminpark2559** [xournalpp/xournalpp#4349]
**Affects versions :**
 - OS: Linux Ubuntu 20.04.5 LTS
 - Which version of libgtk do you use
     - 3.24.20
 - Version of Xournal++: [e.g. 1.0.16, specific commit, specific branch]
     - 1.1.2+dev
 - Installation method: official release

**Describe the bug**
Crash occurs

**To Reproduce**
I think crash happens when I use ctrl+shift+r (selection tool)

**Additional context**
Error screenshot
![image](https://user-images.githubusercontent.com/30585126/197335409-4266533c-1b78-447c-a5e5-0f4fe76f0c61.png)

Error logs
```
Date: Sat Oct 22 19:50:12 2022
Error: signal 11
[bt]: (0) xournalpp(+0xd0a20) [0x557e8c75da20]
[bt]: (1) /lib/x86_64-linux-gnu/libc.so.6(+0x43090) [0x7f0e7fe7a090]
[bt]: (2) xournalpp(_ZN10TextEditor7getTextEv+0x16) [0x557e8c83fec6]
[bt]: (3) xournalpp(_ZN7Control10selectToolE8ToolType+0xfd) [0x557e8c7e2fed]
[bt]: (4) xournalpp(_ZN7Control15actionPerformedE10ActionType11ActionGroupP9_GdkEventP12_GtkMenuItemP14_GtkToolButtonb+0x42f) [0x557e8c7ef86f]
[bt]: (5) xournalpp(_ZN12AbstractItem9activatedEP9_GdkEventP12_GtkMenuItemP14_GtkToolButton+0x82) [0x557e8c85cfb2]
[bt]: (6) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_closure_invoke+0x1b2) [0x7f0e80744802]
[bt]: (7) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(+0x28814) [0x7f0e80758814]
[bt]: (8) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit_valist+0x10ae) [0x7f0e80763bbe]
[bt]: (9) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit+0x93) [0x7f0e807640f3]
[bt]: (10) /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_widget_activate+0x82) [0x7f0e80e95842]
[bt]: (11) xournalpp(_ZN10MainWindow10invokeMenuEP10_GtkWidget+0xd) [0x557e8c799fbd]
[bt]: (12) /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3e81c3) [0x7f0e80ee91c3]
[bt]: (13) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_closure_invoke+0x1b2) [0x7f0e80744802]
[bt]: (14) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(+0x28814) [0x7f0e80758814]
[bt]: (15) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit_valist+0x96d) [0x7f0e8076347d]
[bt]: (16) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit+0x93) [0x7f0e807640f3]
[bt]: (17) /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_accel_group_activate+0xa1) [0x7f0e80c12131]
[bt]: (18) /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_accel_groups_activate+0xd5) [0x7f0e80c13a85]
[bt]: (19) /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_window_activate_key+0x142) [0x7f0e80eb8302]
[bt]: (20) /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3b75e6) [0x7f0e80eb85e6]
[bt]: (21) /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3e727b) [0x7f0e80ee827b]
[bt]: (22) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_closure_invoke+0x1b2) [0x7f0e80744802]
[bt]: (23) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(+0x27f96) [0x7f0e80757f96]
[bt]: (24) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit_valist+0x96d) [0x7f0e8076347d]
[bt]: (25) /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit+0x93) [0x7f0e807640f3]
[bt]: (26) /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3919b3) [0x7f0e80e929b3]
[bt]: (27) /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x24cf8f) [0x7f0e80d4df8f]
[bt]: (28) /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_main_do_event+0x89b) [0x7f0e80d5018b]
[bt]: (29) /lib/x86_64-linux-gnu/libgdk-3.so.0(+0x3cf69) [0x7f0e80a38f69]
[bt]: (30) /lib/x86_64-linux-gnu/libgdk-3.so.0(+0x700f6) [0x7f0e80a6c0f6]
[bt]: (31) /lib/x86_64-linux-gnu/libglib-2.0.so.0(g_main_context_dispatch+0x27d) [0x7f0e8065917d]
[bt]: (32) /lib/x86_64-linux-gnu/libglib-2.0.so.0(+0x52400) [0x7f0e80659400]
[bt]: (33) /lib/x86_64-linux-gnu/libglib-2.0.so.0(g_main_context_iteration+0x33) [0x7f0e806594a3]
[bt]: (34) /lib/x86_64-linux-gnu/libgio-2.0.so.0(g_application_run+0x205) [0x7f0e8089afe5]
[bt]: (35) xournalpp(_ZN11XournalMain3runEiPPc+0x503) [0x557e8c75ede3]
[bt]: (36) /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f0e7fe5b083]
[bt]: (37) xournalpp(_start+0x2e) [0x557e8c75d1be]


Try to get a better stracktrace...
[bt] #1 xournalpp(+0xd0ebc) [0x557e8c75debc]
??:0
[bt] #2 /lib/x86_64-linux-gnu/libc.so.6(+0x43090) [0x7f0e7fe7a090]
??:0
[bt] #3 xournalpp(_ZN10TextEditor7getTextEv+0x16) [0x557e8c83fec6]
??:0
[bt] #4 xournalpp(_ZN7Control10selectToolE8ToolType+0xfd) [0x557e8c7e2fed]
??:0
[bt] #5 xournalpp(_ZN7Control15actionPerformedE10ActionType11ActionGroupP9_GdkEventP12_GtkMenuItemP14_GtkToolButtonb+0x42f) [0x557e8c7ef86f]
??:0
[bt] #6 xournalpp(_ZN12AbstractItem9activatedEP9_GdkEventP12_GtkMenuItemP14_GtkToolButton+0x82) [0x557e8c85cfb2]
??:0
[bt] #7 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_closure_invoke+0x1b2) [0x7f0e80744802]
??:0
[bt] #8 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(+0x28814) [0x7f0e80758814]
??:0
[bt] #9 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit_valist+0x10ae) [0x7f0e80763bbe]
??:0
[bt] #10 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit+0x93) [0x7f0e807640f3]
??:0
[bt] #11 /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_widget_activate+0x82) [0x7f0e80e95842]
??:0
[bt] #12 xournalpp(_ZN10MainWindow10invokeMenuEP10_GtkWidget+0xd) [0x557e8c799fbd]
??:0
[bt] #13 /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3e81c3) [0x7f0e80ee91c3]
??:0
[bt] #14 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_closure_invoke+0x1b2) [0x7f0e80744802]
??:0
[bt] #15 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(+0x28814) [0x7f0e80758814]
??:0
[bt] #16 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit_valist+0x96d) [0x7f0e8076347d]
??:0
[bt] #17 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit+0x93) [0x7f0e807640f3]
??:0
[bt] #18 /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_accel_group_activate+0xa1) [0x7f0e80c12131]
??:0
[bt] #19 /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_accel_groups_activate+0xd5) [0x7f0e80c13a85]
??:0
[bt] #20 /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_window_activate_key+0x142) [0x7f0e80eb8302]
??:0
[bt] #21 /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3b75e6) [0x7f0e80eb85e6]
??:0
[bt] #22 /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3e727b) [0x7f0e80ee827b]
??:0
[bt] #23 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_closure_invoke+0x1b2) [0x7f0e80744802]
??:0
[bt] #24 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(+0x27f96) [0x7f0e80757f96]
??:0
[bt] #25 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit_valist+0x96d) [0x7f0e8076347d]
??:0
[bt] #26 /lib/x86_64-linux-gnu/libgobject-2.0.so.0(g_signal_emit+0x93) [0x7f0e807640f3]
??:0
[bt] #27 /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x3919b3) [0x7f0e80e929b3]
??:0
[bt] #28 /lib/x86_64-linux-gnu/libgtk-3.so.0(+0x24cf8f) [0x7f0e80d4df8f]
??:0
[bt] #29 /lib/x86_64-linux-gnu/libgtk-3.so.0(gtk_main_do_event+0x89b) [0x7f0e80d5018b]
??:0
[bt] #30 /lib/x86_64-linux-gnu/libgdk-3.so.0(+0x3cf69) [0x7f0e80a38f69]
??:0
[bt] #31 /lib/x86_64-linux-gnu/libgdk-3.so.0(+0x700f6) [0x7f0e80a6c0f6]
??:0
```

**Comment by bhennion** [comment]
I think this might be fixed by #4347.

**Comment by youngminpark2559** [comment]
@bhennion Thank you for the check
